### DISCUSSION TITLE
わいわいTech News

### DISCUSSION THREAD

**Discussion by UserSuccessTeamLAPRAS** [lapras-inc/lapras-user-community#16] [General]
# LAPRAS Tech News Talkとは？
毎週月曜日にLAPRASのメンバーが集まってテック系ニュースを中心にYouTube Liveにて情報共有を行っています！🗞

[LAPRAS Tech News Talk](https://www.youtube.com/playlist?list=PLKbaztxP2P4jpdF0P5YbJNJwFabB-pksK) 

# わいわいTech Newsについて
ここでは毎週Tech News Talkの内容のダイジェストを皆様にお届けします 💁‍♀️ 
Tech News Talkのご感想やご自身が共有したいニュースなどぜひお寄せ下さい！

日々技術について調べる中で気になったトピックについて、

- 技術に関心のある誰かに共有したい
- 仕事で使わない技術についてもアンテナを張っていたい
- 「へえ〜」と思ったものの話せる場がない

…など感じたらぜひ投稿してみてください。技術に関するトピックであれば何でも大丈夫です。
（もし、「これは良いのかな？」と迷ったら運営にご相談ください 🙏 ）

**Comment by ktnyt** [comment]
# Mojoを触ってみた/ジェフリー・ヒントンがグーグルを退社/LangChain を利用したシステマティック・レビューを生成するアプリ
今週のTech News Talkが公開されました :tada:
https://www.youtube.com/watch?v=IgJLhiC0a90

## ダイジェスト
今週のテックトークは
- Pythonのスーパーセットである新しいプログラミング言語「Mojo」を使ってみた感触（by Takahashi）
- 深層学習の生みの親の一人、Goeffrey Hinton氏のGoogle退社とAI安全性について（by ことね）
- LLM+LangChainを用いて特定の話題に関する研究のまとめ論文を生成してみた（by Tajima）
の３つのテーマでお送りしました！

## ことねの一言
ハイパフォーマンスなPythonスーパーセットやHinton氏の話題もさることながら、弊社エンジニアのTajimaさんが開発されたシステマティック・レビュー生成は画期的でした！興味があるけどどこからインプットしたらいいかわからない時に大活躍しそうな予感！？

**Comment by ytimesy** [comment]
人を癒すためのAIアプリを開発中
まるで人と話をしているような感覚になるようなWebアプリです。
地球上のすべての人に使って欲しいです。

**Reply by ktnyt** [reply]
素敵なアプリケーションを開発されているんですね！ぜひがんばってください！

**Comment by ktnyt** [comment]
# スクフェス新潟2023/素朴理論、建設的相互作用/ISUCON13開催決定
今週のTech News Talkが公開されました :tada:
https://www.youtube.com/watch?v=pdmG02zhScw

## ダイジェスト
今週のテックトークでは
- アジャイルコミュニティの祭典！スクラムフェス新潟2023の登壇記（by ことね）
- 経験から理解に繋げる重要性とペア作業が理解に与える効果について（by endo）
- ISUCON2023開催決定！今年は本選一発勝負！（by takahashi）
の３つのテーマでお送りしました！

## ことねの一言
僭越ながら登壇してきた内容を共有させていただきました！教育心理学的アプローチから見たペアプロの効能の話はとても納得感がありましたね。

**Comment by ktnyt** [comment]
# スマート歯ブラシをハックしてみた/MITで60年間教えていた名物教授、ギルバート・ストラング博士の最後の授業
今週のTech News Talkが公開されています :クラッカー:
https://www.youtube.com/watch?v=lbDQrEOkluY

## ダイジェスト
- スマート歯ブラシをハックしてどういうデータをどのように使っているのかを調べた話
- 「線形代数を楽しくした」と評価されたMIT教授の軌跡の話

## ことねの一言
UXを損なわないように最小労力で磨耗状況などを収集する知恵の絞り方がすごいですね。ストラング博士は66年間もMITに在籍していて親子で同じ授業を受けた可能性もあるというのは滅多にない功績ですね。
